coderabbit.yml

```
@@ -0,0 +1,71 @@
+      - Review for performance optimizations
+
+  - path: "package.json"
+    instructions: |
+      Review dependency management:
+      - Check for security vulnerabilities in dependencies
+      - Ensure version pinning where appropriate
+      - Verify scripts are properly configured
+
+# Ignore specific patterns
+ignored_titles: "WIP,Draft,Test"
+ignored_branch: "dev/test,experimental/*" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)